perma_web/perma/tasks.py

```diff
@@ -440,7 +462,7 @@ def get_meta():
                 }
                 return tags
             """)
-    return repeat_until_truthy(get_meta)
+    return repeat_while_exception(lambda: repeat_until_truthy(get_meta), exception=HTTPException)
 
```

Comment:
Aha, thanks for including these fixes too. I still haven't tested to see what happens w/o repeat_until_truthy; will do shortly. Won't delay merging this to wait for that!